docker/GUNC/filter.sh

```diff
@@ -1,30 +1,35 @@
 #!/bin/bash
 
-while getopts :c:g: option; do
+# Input GUNC file should have 2 lines: header+genome annotation
+# Because we are running GUNC on each genome separately in pipeline
+while getopts :c:g:n: option; do
 	case "${option}" in
         c) CSV=${OPTARG};;
         g) GUNC=${OPTARG};;
+        n) NAME=${OPTARG};;
 	esac
 done
 
-
+### check GUNC
+# gunc contaminated genomes
 awk '{if($8 > 0.45 && $9 > 0.05 && $12 > 0.5)print$1}' ${GUNC} | grep -v "pass.GUNC" > gunc_contaminated.txt
```

Comment:
About using double quotes in variables: https://github.com/koalaman/shellcheck/wiki/SC2086

```bash
${GUNC} => "${GUNC}"
```
I don't think this would be a problem here, but it's a good practice